sklearn/utils/tests/test_linear_assignment.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
```

Comment:
we don't need this file. It can be removed entirely. Scipy tests their implementation.

Reply:
Actually let's leave it in to maintain code coverage. Use the version in master, only changing it to ignore deprecation warnings during import. That way we will be sure to remove it in 0.23.

Reply:
done